### ISSUE TITLE
Fix for compatibility with dearpygui version 1.11.1

### ISSUE THREAD

**Issue by TorgovetsSKotom** [Kazuhito00/Image-Processing-Node-Editor#20]
The following changes have been made to correct:

- For *_callback_link* **data** is no list of **tag**'s, but a list of **id**'s, so:
   - **data[0]** wrapped in *dpg.get_item_alias(data[0])*
   - Removed *join* for debug print

- In *_sort_node_graph* **node_link_info** is also a list of **id**'s, because **source** and **destination** are placed at the beginning of the loop and wrap **node_link_info[0]** using *dpg.get_item_alias(node_link_info[0])*

- Deleted *delink_callback* due to inactivity. In connection with https://github.com/hoffstadt/DearPyGui/issues/1746 this was fixed in DearPyGui version 1.6.2, but at the moment it either does not work, or its use is not obvious.
To replace in *_callback_mv_key_del* added:

```python
if len(dpg.get_selected_links(self._node_editor_tag)) > 0:
    self._node_link_list.remove([
        dpg.get_item_alias(dpg.get_item_configuration(dpg.get_selected_links(self._node_editor_tag)[0])['attr_1']),
        dpg.get_item_alias(dpg.get_item_configuration(dpg.get_selected_links(self._node_editor_tag)[0])['attr_2'])
    ])
    self._node_connection_dict = self._sort_node_graph(
        self._node_list,
        self._node_link_list,
    )
    dpg.delete_item(dpg.get_selected_links(self._node_editor_tag)[0])
```

Notes:
To run the latest version from the repository, I had to disable node_qrcode_detection.py .
Also, connecting BRISQUE breaks the video operation.

**Comment by TorgovetsSKotom** [comment]
fix 985d6915664c370acbed8aa8cfe29d12925fc1da

**Comment by Kazuhito00** [comment]
Thank you for the PR. Now works with the latest version of DearPyGUI.
https://github.com/Kazuhito00/Image-Processing-Node-Editor/pull/21
